portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,73 @@
 <!DOCTYPE html>
 <html>
+    
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <!-- <style>
+         body {
+            background-image: url("/images.JPG");
+         }
+      </style> -->
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <!-- <img class = bgImage src = "/images/bg2.JPG"> -->
+       
+      <button id="play_button" onclick="loadIntro() ">Cynthia Ma</button>
```

Comment:
This should be dedented one level to align with other elements directly in the body.